extensions/interactions/DragAndDropSortInput/directives/drag-and-drop-sort-input-response.component.html

```diff
@@ -34,7 +34,7 @@
 
 <ul *ngIf="isAnswerLengthGreaterThanZero" class="drag-and-drop-sort-response-content">
   <li *ngFor="let responses of responseList">
-    <ul [(ngModel)]="responses">
+    <ul [(ngModel)]="responses" ngDefaultControl>
```

Comment:
Can you explain this change please?

Reply:
This is unrelated to the actual issue -- the line was causing the following console error:

```
No value accessor for form control with unspecified name attribute
```

From what I've found online, the `ngModel` directive is only allowed on elements that have a value attribute, but adding the `ngDefaultControl` keyword allows it to be used on other elements (such as this ul tag which lacks a value accessor) as well.

Reply:
Thanks perhaps adding a comment here might be useful so other developers looking at this will know what its for.

Reply:
Done!